avro/src/main/scala/hydra/avro/registry/RedisSchemaRegistryClient.scala

```diff
@@ -280,19 +340,50 @@ class RedisSchemaRegistryClient(restService: RestService,
     restService.getAllSubjectsById(i)
   }
 
-  override def getLatestSchemaMetadata(s: String): SchemaMetadata = synchronized {
+   def getLatestSchemaMetadata(s: String): SchemaMetadata = synchronized {
```

Comment:
The same as above about `synchronized`

Reply:
this one was add by mistake and I will remove it

Reply:
this one was added based on the implementation in the io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient. The CachedSchemaRegistryClient was used as a source of implementation.